server/resources/[VORP]/[vorp_essentials]/vorp_core/server/sv_apicontroller.lua

```diff
@@ -80,8 +80,10 @@ AddEventHandler('vorp:setJob', function(player, job, jobgrade)
     local used_char = _getUsedCharacter(player)
 
     if used_char ~= nil then
-        used_char.setJob(job)
-        used_char.setJobGrade(jobgrade)
+        if validateJob(job,jobgrade) then
```

Comment:
and what should happen if validatejob fails? Also I feel like this requires a callback to properly sync up to user